app/src/main/res/drawable/ic_password.xml

```diff
@@ -5,5 +5,5 @@
         android:viewportHeight="24.0">
     <path
         android:pathData="M18,8h-1L17,6c0,-2.76 -2.24,-5 -5,-5S7,3.24 7,6v2L6,8c-1.1,0 -2,0.9 -2,2v10c0,1.1 0.9,2 2,2h12c1.1,0 2,-0.9 2,-2L20,10c0,-1.1 -0.9,-2 -2,-2zM12,17c-1.1,0 -2,-0.9 -2,-2s0.9,-2 2,-2 2,0.9 2,2 -0.9,2 -2,2zM15.1,8L8.9,8L8.9,6c0,-1.71 1.39,-3.1 3.1,-3.1 1.71,0 3.1,1.39 3.1,3.1v2z"
-        android:fillColor="#555"/>
+        android:fillColor="#2196f3"/>
```

Comment:
Please don't change the default color of drawables. It is being used throughout the codebase.

Reply:
@Masquerade0097 When I clicked on 'find usages' I see no location in which the file is used, hence I changed the color. As I am new to this project please tell me where I m wrong.

Reply:
`ic_password` is not being used anywhere but most of the drawables have the default color `#555` so rather than changing the default color, you can tint the drawable to a for the layout.